cpp/include/cudf_test/column_wrapper.hpp

```diff
@@ -1820,7 +1820,7 @@ class structs_column_wrapper : public detail::column_wrapper {
    * child_columns.push_back(std::move(child_int_col));
    * child_columns.push_back(std::move(child_string_col));
    *
-   * struct_column_wrapper struct_column_wrapper{
+   * structs_column_wrapper struct_column_wrapper{
```

Comment:
The type and name are easy to misunderstand here. Maybe we can rename the variable.
```suggestion
   * structs_column_wrapper structs_col{
```

Reply:
renamed.